recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -78,65 +76,80 @@ export default {
       default: () => '',
     },
 
-    /**
-     * Shows/hides the expanded portion
-     */
-    expanded: {
-      type: Boolean,
-      default: () => false,
-    },
-
     /**
      * Additional styling around the pill
      */
     wrapperClass: {
-      type: String,
-      default: () => '',
+      type: [String, Array, Object],
+      default: '',
     },
 
     /**
      * Additional styling for the pill
      */
     buttonClass: {
-      type: String,
-      default: () => '',
+      type: [String, Array, Object],
+      default: '',
     },
 
     /**
      * Aria label for feed pill
      */
     ariaLabel: {
       type: String,
-      default: () => '',
+      default: null,
+      validator: (val) => {
+        return !!val;
+      },
     },
 
     /**
-     * Sets whether the pill can be toggled
+     * Sets whether the pill can be toggled (Icon changing on hover, expanding and collapsing, pointer)
      */
     toggleable: {
       type: Boolean,
       default: () => true,
     },
   },
 
-  emits: [
-    'click',
-  ],
```

Comment:
Isn't this event required for the component? It wouldn't hurt to have this event emitted and it could potentially be used for something else by the consumer.

Reply:
Should be no cases where the pill does anything aside from expand